<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1388</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>1000</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="calculatePushButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>170</y>
         <width>141</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Рассчитать</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>230</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="F0_doubleSpinBox">
          <property name="minimum">
           <double>-1000000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="value">
           <double>50.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="F0_units_label">
          <property name="text">
           <string>Вт/см&lt;sup&gt;2&lt;/sup&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="F0_label">
          <property name="text">
           <string>F&lt;sub&gt;0&lt;/sub&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QCustomPlot" name="Tx_customPlot" native="true"/>
    </item>
    <item>
     <widget class="QCustomPlot" name="Tt_customPlot" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
